src/Symfony/Component/HttpKernel/Debug/TraceableEventDispatcher.php

```diff
@@ -61,7 +61,7 @@ protected function preDispatch($eventName, Event $event)
     protected function postDispatch($eventName, Event $event)
     {
         switch ($eventName) {
-            case KernelEvents::CONTROLLER:
+            case KernelEvents::CONTROLLER_ARGUMENTS:
```

Comment:
@Taluu this is about starting the stopwatch for the controller execution itself. As there is now an event after the resolution of arguments, I starting it there now, as it makes the timing more accurate (in existing versions, the argument resolution is included in the controller time, even though it is also timed on its own)